### ISSUE TITLE
Permanent link to most recent OTT

### ISSUE THREAD

**Issue by josephwb** [OpenTreeOfLife/reference-taxonomy#74]
It would be nice to have a permanent link to the most recent version of OTT. For example, I want the most recent version, but don't know what that version might be (and, thus, not be able to guess the URL). I ask because we are thinking of scripts that would automatically obtain and prune the taxonomy for a user's particular clade of interest. Of course, the user could just download OTT and prune it themselves, but we are kicking around the idea of automating it.

**Comment by hyanwong** [comment]
Yes. I also think it would be useful for the files themselves to have (say) 5.0 in the name: if I have a couple of these versions lying around, it's handy to know which ones I'm using. Then the symlink would be to the file, without the 5.0 (or whatever) in the name. That's how wikipedia do it (e.g. see https://dumps.wikimedia.org/enwiki/20160407/ and equivalent https://dumps.wikimedia.org/enwiki/latest/), but that's much more complicated for you to implement than simply symlinking the most recent directory, so you might not want to do that.

**Comment by jar398** [comment]
Check what's here now and let me know if it is what you are looking for:

If these looks good I can write a script to make a new version current.
http://files.opentreeoflife.org/synthesis/current/
http://files.opentreeoflife.org/synthesis/opentree5.0/

When you unpack the tarball, there's going to be a version number (not
'current') in the top level directory name; I'd rather not change make a
separate tarball for the current version. But you could write a script that
figures out the name and symlinks a local 'current' to it.

(I also added the tree file with taxon names to the tarball. Maybe next
time around I'll remove the one without taxon names, to make the tarball
smaller...)

**Comment by hyanwong** [comment]
Thanks for the quick work. It's certainly easier like this. I wonder if it would be sensible to include the extra symlink in the top level dir, so that a bespoke script wouldn't be needed to get to the most recent version after download (helps with automation). I guess that would be overwritten if you then download OTv6.0, etc etc, so the link would point to the most recently downloaded version - perhaps it should just be called 'opentree', since it's not guaranteed to be the current one, if you e.g. download v6 then v5.

**Comment by jar398** [comment]
It is hardly even a script; just a somewhat longish command.

    mkdir unpack && (cd unpack && tar xzvf ../file.tgz) && mv unpack/* current && rmdir unpack

I'm not keen on putting a 'current' link inside a tarball; I think it would make a confusing situation even more confusing.

**Comment by hyanwong** [comment]
Yes, I think 'current' would make it confusing. 'OpenTree' less so. But as you say, simpler without. I just think making it slick to move to a new tree would be helpful.
